docs/releases/3.0.0.rst

```diff
@@ -0,0 +1,68 @@
+Translate Toolkit 3.0.0
+***********************
+
+*Released on 15 June 2020*
+
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- l20n
+
+    - Dropped support for deprecated format
+
+- Windows RC
+
+    - Rewritten parser using pyparsing
+
+- JSON
+
+    - Added support for ARB files
+    - Added support for go-i18n files
+
+- Properties
+
+    - Added support for GWT personality
+    - Fix round trip of empty values without delimiter
+
+- PO
+
+    - Bring line wrapping closer to gettext
+
+- HTML
+
+    - A makeover of the HTML parsing to fix several issues
+
+- PHP
+
+    - Add support for Laravel plurals
+    - Improve round trip of some statements
+
+- XLIFF 
+
+    - Support non numeric ids on plurals in poxliff
+
```

Comment:
These notes are missing the removal of deprecated code like `segment` module, and also https://github.com/translate/translate/pull/4045#pullrequestreview-428678746

Reply:
Yes, I forgot these. Adding them now.